src/Apps/IdentityVerification/routes.tsx

```diff
@@ -2,10 +2,19 @@ import loadable from "@loadable/component"
 import { RouteConfig } from "found"
 import { graphql } from "react-relay"
 
+const Processing = loadable(() => import("./Processing"))
+
 export const routes: RouteConfig[] = [
+  {
+    path: "/identity-verification/processing",
+    getComponent: () => Processing,
+  },
   {
     path: "/identity-verification/:id",
     getComponent: () => loadable(() => import("./IdentityVerificationApp")),
+    prepare: () => {
+      Processing.preload()
+    },
```

Comment:
Intentional to preload `Processing` component rather than `IdentityVerificationApp`?